wrapper.go

```diff
@@ -48,15 +50,31 @@ func (c *commandError) Stderr() string {
 	return c.stderr
 }
 
+func (c *commandError) ExitCode() (int, error) {
+	if exitError, ok := asExitError(c.err); ok {
+		return exitError.ExitCode(), nil
+	} else {
+		return 0, errors.New("not and exit error. Exit code not available")
```

Comment:
typo: "not an exit error"
(I know I'm a typo master myself!)

Reply:
Think can be shortened to "exit code not available"